### ISSUE TITLE
Can't distinguish faces with existing models

### ISSUE THREAD

**Issue by sunjieseu** [ox-vgg/vgg_face2#22]
Hi All，
 MainModel = imp.load_source('MainModel','resnet50_scratch_pytorch.py')
 model = torch.load('resnet50_scratch_pytorch.pth')
 model.eval()   
img_path = ‘the picture 's path and name’
img = Image.open(img_path).convert("RGB")
img = img.resize((224, 224),Image.ANTIALIAS)
input = transforms(img).unsqueeze_(0)
output = model(input).detach().to('cpu').numpy()
feature = output.flatten()

When I input  2 different faces, the features obtained were very close and could not be distinguished.
 what should I do to use the pre-trained model correctly?

Thanks.

**Comment by sunjieseu** [comment]
I have solved this issue.
Added the following 2 sentences and then sent to the model input.

 to_tensor = torchvision.transforms.ToTensor()
 input = to_tensor(rotate_channels(img))*255 - torch.Tensor([91.4953, 103.8827, 131.0912]).view((3,1,1)) 

The model is available, thanks to the original author.

**Comment by WeidiXie** [comment]
I will close this issue then.

Best,
Weidi

**Comment by mumuyanyan** [comment]
> I have solved this issue.
> Added the following 2 sentences and then sent to the model input.
> 
> to_tensor = torchvision.transforms.ToTensor()
> input = to_tensor(rotate_channels(img))*255 - torch.Tensor([91.4953, 103.8827, 131.0912]).view((3,1,1))
> 
> The model is available, thanks to the original author.

why?
